third_party/move/move-prover/lab/src/benchmark.rs

```diff
@@ -149,27 +143,9 @@ fn run_benchmark(
     } else {
         Options::default()
     };
-    let addrs = parse_addresses_from_options(options.move_named_address_values.clone())?;
+    options.move_sources.append(&mut modules.to_vec());
     options.move_deps.append(&mut dep_dirs.to_vec());
-    let skip_attribute_checks = true;
-    let known_attributes = KnownAttribute::get_all_attribute_names().clone();
-    let mut env = run_model_builder_with_options(
-        vec![PackagePaths {
-            name: None,
-            paths: modules.to_vec(),
-            named_address_map: addrs.clone(),
-        }],
-        vec![],
-        vec![PackagePaths {
-            name: None,
-            paths: options.move_deps.clone(),
-            named_address_map: addrs,
-        }],
-        options.model_builder.clone(),
-        skip_attribute_checks,
-        &known_attributes,
-    )?;
-    let mut error_writer = StandardStream::stderr(ColorChoice::Auto);
+    options.skip_attribute_checks = true;
 
     if use_aptos_natives {
         options.backend.custom_natives =
```

Comment:
Again, not related directly to this PR, but it looks like we depend on non-third-party code here (`../../../../../aptos-move/framework/src/aptos-natives.bpl`) within third-party?